catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -92,6 +97,24 @@ public CswRecordCollection readFrom(Class<CswRecordCollection> type, Type generi
             MultivaluedMap<String, String> httpHeaders, InputStream inStream)
             throws IOException, WebApplicationException {
 
+        CswRecordCollection cswRecords = null;
+
+        // If the following HTTP header exists and its value is true, the input stream will contain
+        // raw product data
+        List<String> productRetrievalHeader =
+                httpHeaders.get(CswConstants.PRODUCT_RETRIEVAL_HTTP_HEADER);
+        if (productRetrievalHeader != null) {
+            if (productRetrievalHeader.get(0)
```

Comment:
Also, couldn't the string be "True" or "TRUE"? Maybe use `equalsIgnoreCase` or convert the value to a boolean and test it?